### PR TITLE
Convert all orientation operations to double precision where possible

### PR DESCRIPTION
**Pull request by imikejackson**
All LaueOps classes and subclasses have been converted to use double precision floating point where practically possible.

### CODE REVIEW

File: Source/OrientationLib/LaueOps/OrthoRhombicOps.cpp [hunk 1]
```diff
@@ -956,10 +966,10 @@ UInt8ArrayType::Pointer OrthoRhombicOps::generateIPFTriangleLegend(int imageDim)
       x = xIndex * xInc;
       y = yIndex * yInc;
 
-      float sumSquares = (x * x) + (y * y);
-      if( sumSquares > 1.0f) // Outside unit circle
+      double sumSquares = (x * x) + (y * y);
+      if(sumSquares > 1.0) // Outside unit circle
       {
-        color = 0xFFFFFFFF;
+        color = 0xFFFFFFF;
```
Comment (mmarineBlueQuartz) [inline Source/OrientationLib/LaueOps/OrthoRhombicOps.cpp]: Malformed hex color.  You need that missing F for an even number forming ARGB / RGBA or to remove another one for RGB.

File: Source/Plugins/Reconstruction/ReconstructionFilters/MergeColonies.cpp [hunk 2]
```diff
@@ -399,58 +400,59 @@ int32_t MergeColonies::getSeed(int32_t newFid)
...
+      float angdiff2 = std::fabs(w - 90.00f);
+      float axisdiff2 = std::acos(std::fabs(n1) * 0.9958f + std::fabs(n2) * 0.0917f + std::fabs(n3) * 0.0000f);
       if(angdiff2 < m_AngleTolerance && axisdiff2 < m_AxisToleranceRad)
       {
         colony = true;
       }
-      float angdiff3 = fabsf(w - 60.00f);
-      float axisdiff3 = acosf(fabsf(n1) * 1.0000f + fabsf(n2) * 0.0000f + fabsf(n3) * 0.0000f);
+      float angdiff3 = std::fabs(w - 60.00f);
+      float axisdiff3 = std::acos(std::fabs(n1) * 1.0000f + std::fabs(n2) * 0.0000f + std::fabs(n3) * 0.0000f);
```
Comment (mmarineBlueQuartz) [inline Source/Plugins/Reconstruction/ReconstructionFilters/MergeColonies.cpp]: Why are we spending processing power on std::fabs(n3) and std::fabs(n2) when we're multiplying by 0?

Comment (imikejackson) [reply Source/Plugins/Reconstruction/ReconstructionFilters/MergeColonies.cpp]: ummmm... got nothing ....

Comment (imikejackson) [reply Source/Plugins/Reconstruction/ReconstructionFilters/MergeColonies.cpp]: git log + git blame...

**Pull request merged**